assets/cluster-version-operator/cluster-version-operator-deployment.yaml

```diff
@@ -30,6 +30,10 @@ spec:
                       values: ["{{ .ClusterID }}"]
                 topologyKey: "kubernetes.io/hostname"
       tolerations:
+        - key: "dedicated"
+          operator: "Equal"
+          value: "master-{{ cluster_id }}"
```

Comment:
Please replace `cluster_id` with `.ClusterID`.  This is a general comment that applies to this entire PR.

Reply:
Updated. Thank you